build.gradle

```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.2.21'
+    ext.kotlin_version = '1.2.41'
```

Comment:
can we remove the kotlin dependecy ?
It`s only used for an empty MainActivity ;)

Reply:
Sure, I'm also going to delete the sample app for now as it doesn't do anything anyway.